### PR TITLE
Modify nvmeof sanity test suite with hostnqn addition for host add to subsystem

### PR DESCRIPTION
**Pull request by rahullepakshi**
Added support for subsystem access through hostnqn of initiator by provisioning hostnqn to host add command

```
        install: true                           # Run SPDK with all pre-requisites
        cleanup:
          - subsystems
          - initiators
          - pool
          - gateway
        subsystems:                             # Configure subsystems with all sub-entities
          - nqn: nqn.2016-06.io.spdk:cnode1
            serial: 1
            bdevs:
              count: 1
              size: 10G
            listener_port: 5001
            initiator_node: node10

```

```
2024-08-01 14:03:51,251 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : subsystem add
2024-08-01 14:03:51,251 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm quay.io/ceph/nvmeof-cli:latest  --server-address 10.0.65.76 --server-port 5500 subsystem add  --subsystem nqn.2016-06.io.spdk:cnode1 --max-namespaces 32 on 10.0.65.76 timeout 600
2024-08-01 14:03:51,881 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-01 14:03:51,881 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Adding subsystem nqn.2016-06.io.spdk:cnode1: Successful\n')
2024-08-01 14:03:51,881 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : gw info
2024-08-01 14:03:51,881 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm quay.io/ceph/nvmeof-cli:latest  --format json --server-address 10.0.65.76 --server-port 5500 gw info  on 10.0.65.76 timeout 600
2024-08-01 14:03:52,204 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command modprobe nvme-fabrics on 10.0.64.95 timeout 600
2024-08-01 14:03:52,222 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command nvme show-hostnqn on 10.0.64.95 timeout 600
2024-08-01 14:03:52,241 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-01 14:03:52,241 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : subsystem add
2024-08-01 14:03:52,242 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm quay.io/ceph/nvmeof-cli:latest  --server-address 10.0.65.76 --server-port 5500 subsystem add  --subsystem nqn.2016-06.io.spdk:cnode2 --max-namespaces 32 on 10.0.65.76 timeout 600
2024-08-01 14:03:52,477 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-01 14:03:52,477 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', '{\n    "cli_version": "1.2.17",\n    "version": "1.2.17",\n    "name": "client.nvmeof.rbd.ceph-nvme-1-45d5ez-node6.mzxgwu",\n    "addr": "10.0.65.76",\n    "port": "5500",\n    "bool_status": true,\n    "error_message": "Success",\n    "spdk_version": "24.01",\n    "load_balancing_group": 1,\n    "hostname": "ceph-nvme-1-45d5ez-node6",\n    "group": "",\n    "status": 0\n}\n')
2024-08-01 14:03:52,478 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : listener add
2024-08-01 14:03:52,478 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm quay.io/ceph/nvmeof-cli:latest  --server-address 10.0.65.76 --server-port 5500 listener add  --host-name ceph-nvme-1-45d5ez-node6 --traddr 10.0.65.76 --trsvcid 5001 --subsystem nqn.2016-06.io.spdk:cnode1 on 10.0.65.76 timeout 600
2024-08-01 14:03:52,778 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-01 14:03:52,778 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Adding subsystem nqn.2016-06.io.spdk:cnode2: Successful\n')
2024-08-01 14:03:52,778 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : gw info
2024-08-01 14:03:52,779 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm quay.io/ceph/nvmeof-cli:latest  --format json --server-address 10.0.65.76 --server-port 5500 gw info  on 10.0.65.76 timeout 600
2024-08-01 14:03:53,031 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-01 14:03:53,031 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Adding nqn.2016-06.io.spdk:cnode1 listener at 10.0.65.76:5001: Successful\n')
2024-08-01 14:03:53,031 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : host add
2024-08-01 14:03:53,031 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm quay.io/ceph/nvmeof-cli:latest  --server-address 10.0.65.76 --server-port 5500 host add  --subsystem nqn.2016-06.io.spdk:cnode1 --host nqn.2014-08.org.nvmexpress:uuid:db2db797-95a8-4655-a87b-31f59d416008 on 10.0.65.76 timeout 600
2024-08-01 14:03:53,299 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-01 14:03:53,300 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', '{\n    "cli_version": "1.2.17",\n    "version": "1.2.17",\n    "name": "client.nvmeof.rbd.ceph-nvme-1-45d5ez-node6.mzxgwu",\n    "addr": "10.0.65.76",\n    "port": "5500",\n    "bool_status": true,\n    "error_message": "Success",\n    "spdk_version": "24.01",\n    "load_balancing_group": 1,\n    "hostname": "ceph-nvme-1-45d5ez-node6",\n    "group": "",\n    "status": 0\n}\n')
2024-08-01 14:03:53,300 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : listener add
2024-08-01 14:03:53,300 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm quay.io/ceph/nvmeof-cli:latest  --server-address 10.0.65.76 --server-port 5500 listener add  --host-name ceph-nvme-1-45d5ez-node6 --traddr 10.0.65.76 --trsvcid 5002 --subsystem nqn.2016-06.io.spdk:cnode2 on 10.0.65.76 timeout 600
2024-08-01 14:03:53,567 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-01 14:03:53,567 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Adding host nqn.2014-08.org.nvmexpress:uuid:db2db797-95a8-4655-a87b-31f59d416008 to nqn.2016-06.io.spdk:cnode1: Successful\n')
2024-08-01 14:03:53,955 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-01 14:03:53,955 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Adding nqn.2016-06.io.spdk:cnode2 listener at 10.0.65.76:5002: Successful\n')
2024-08-01 14:03:53,955 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : host add
2024-08-01 14:03:53,955 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm quay.io/ceph/nvmeof-cli:latest  --server-address 10.0.65.76 --server-port 5500 host add  --subsystem nqn.2016-06.io.spdk:cnode2 --host nqn.2014-08.org.nvmexpress:uuid:db2db797-95a8-4655-a87b-31f59d416008 on 10.0.65.76 timeout 600
2024-08-01 14:03:54,605 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-01 14:03:54,606 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Adding host nqn.2014-08.org.nvmexpress:uuid:db2db797-95a8-4655-a87b-31f59d416008 to nqn.2016-06.io.spdk:cnode2: Successful\n')
2024-08-01 14:03:54,607 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman --version | awk {'print $3'} on 10.0.66.187 timeout 600
2024-08-01 14:03:54,646 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.:1053 - Podman Version 4.6.1
2024-08-01 14:03:54,646 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command docker --version | awk {'print $3'} on 10.0.66.187 timeout 600
2024-08-01 14:03:54,704 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command ceph --version | awk '{print $3}' on 10.0.64.95 timeout 600
2024-08-01 14:03:54,805 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.:1069 - ceph Version 19.0.0-5313-ge5d33fa4
2024-08-01 14:03:54,805 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.:919 - Test <module 'test_ceph_nvmeof_gateway' from '/root/new1/cephci/tests/nvmeof/test_ceph_nvmeof_gateway.py'> passed
Test <module 'test_ceph_nvmeof_gateway' from '/root/new1/cephci/tests/nvmeof/test_ceph_nvmeof_gateway.py'> passed
All test logs located here: /tmp/cephci-run-N8PTDH
```

### CODE REVIEW

File: suites/reef/nvmeof/tier-1_nvmeof_sanity.yaml [hunk 1]
```diff
@@ -122,7 +122,7 @@ tests:
               count: 1
               size: 10G
             listener_port: 5001
-            allow_host: "*"
+            initiator_node: node10
```
Comment (sunilkumarn417) [inline suites/reef/nvmeof/tier-1_nvmeof_sanity.yaml:125]: A suggestion, make it list, so it would accomodate many host.

Comment (sunilkumarn417) [reply suites/reef/nvmeof/tier-1_nvmeof_sanity.yaml:125]: Keep it as `hosts: []`.

**Comment by rahullepakshi** [comment]
@sunilkumarn417 I tested to add with 2 initiators as below

```
        subsystems:                             # Configure subsystems with all sub-entities
          - nqn: nqn.2016-06.io.spdk:cnode1
            serial: 1
            bdevs:
              count: 1
              size: 10G
            listener_port: 5001
            hosts:
              - node10
              - node2
```

results are below

```
2024-08-06 13:56:19,272 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : listener add
2024-08-06 13:56:19,272 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.2.17-8  --server-address 10.0.65.219 --server-port 5500 listener add  --host-name ceph-nvme-1-wzk17m-node6 --traddr 10.0.65.219 --trsvcid 5001 --subsystem nqn.2016-06.io.spdk:cnode1 on 10.0.65.219 timeout 600
2024-08-06 13:56:20,245 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-06 13:56:20,245 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Adding nqn.2016-06.io.spdk:cnode1 listener at 10.0.65.219:5001: Successful\n')
**2024-08-06 13:56:20,246 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command yum install -y nvme-cli fio on 10.0.64.123 timeout 600
2024-08-06 13:56:22,147 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command modprobe nvme-fabrics on 10.0.64.123 timeout 600
2024-08-06 13:56:22,205 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command nvme show-hostnqn on 10.0.64.123 timeout 600
2024-08-06 13:56:22,223 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-06 13:56:22,223 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : host add
2024-08-06 13:56:22,224 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.2.17-8  --server-address 10.0.65.219 --server-port 5500 host add  --subsystem nqn.2016-06.io.spdk:cnode1 --host nqn.2014-08.org.nvmexpress:uuid:a92ca03f-7c47-4940-adc2-483468d0326c on 10.0.65.219 timeout 600
2024-08-06 13:56:23,133 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-06 13:56:23,133 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Adding host nqn.2014-08.org.nvmexpress:uuid:a92ca03f-7c47-4940-adc2-483468d0326c to nqn.2016-06.io.spdk:cnode1: Successful\n')
2024-08-06 13:56:23,133 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command yum install -y nvme-cli fio on 10.0.67.93 timeout 600
2024-08-06 13:56:35,815 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command modprobe nvme-fabrics on 10.0.67.93 timeout 600
2024-08-06 13:56:35,931 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command nvme show-hostnqn on 10.0.67.93 timeout 600
2024-08-06 13:56:35,977 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-06 13:56:35,977 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : host add
2024-08-06 13:56:35,977 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.2.17-8  --server-address 10.0.65.219 --server-port 5500 host add  --subsystem nqn.2016-06.io.spdk:cnode1 --host nqn.2014-08.org.nvmexpress:uuid:99bea592-adea-4409-95ff-0f5f2373e431 on 10.0.65.219 timeout 600
2024-08-06 13:56:36,878 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-06 13:56:36,879 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Adding host nqn.2014-08.org.nvmexpress:uuid:99bea592-adea-4409-95ff-0f5f2373e431 to nqn.2016-06.io.spdk:cnode1: Successful\n')**
2024-08-06 13:56:36,879 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command rbd create rbd/XMT5-image0 --size 10G on 10.0.64.123 timeout 600
2024-08-06 13:56:37,002 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-06 13:56:37,002 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.rbd.rbd_utils.py:107 - Command execution complete
2024-08-06 13:56:37,002 (cephci.test_ceph_nvmeof_gateway) [DEBUG] - cephci.parallel.py:89 - result is 0
2024-08-06 13:56:37,003 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : namespace add
2024-08-06 13:56:37,003 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.2.17-8  --server-address 10.0.65.219 --server-port 5500 namespace add  --subsystem nqn.2016-06.io.spdk:cnode1 --rbd-pool rbd --rbd-image XMT5-image0 on 10.0.65.219 timeout 600
2024-08-06 13:56:37,965 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-06 13:56:37,966 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Adding namespace 1 to nqn.2016-06.io.spdk:cnode1: Successful\n')
```

File: suites/reef/nvmeof/tier-1_nvmeof_sanity.yaml (hunk 1 above)
Comment (rahullepakshi) [reply suites/reef/nvmeof/tier-1_nvmeof_sanity.yaml]: Done

File: suites/reef/nvmeof/tier-1_nvmeof_sanity.yaml [hunk 2]
```diff
@@ -122,7 +122,8 @@ tests:
               count: 1
               size: 10G
             listener_port: 5001
-            allow_host: "*"
+            hosts:
```
Comment (sunilkumarn417) [inline suites/reef/nvmeof/tier-1_nvmeof_sanity.yaml:125]: Support both "*" and list of nodes. If we continue to use `hosts`, probably check out the other test suites which needs changes.
or go with allow_host to support only '*' with if condition.

Comment (rahullepakshi) [reply suites/reef/nvmeof/tier-1_nvmeof_sanity.yaml:132]: @sunilkumarn417 please check now per log below

```
2024-08-07 07:37:26,629 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1528 - long running command on 10.0.65.219 -- podman login --username cp --password eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJJQk0gTWFya2V0cGxhY2UiLCJlbnYiOiJzdGFnZSIsImlhdCI6MTcxMzc3NTQ1NSwianRpIjoiYzZmMTc5N2JiOTA4NGQ2YzkwZWUxNjQ3MzNhOWFmNDAifQ.DgpzR7VvRQ3ZUFwD_CCCH57gy3wuv_BRvva7F_sXCjE cp.stg.icr.io with 3600 seconds
2024-08-07 07:37:27,633 (cephci.test_ceph_nvmeof_gateway) [DEBUG] - cephci.ceph.py:1553 - b'Login Succeeded!'
2024-08-07 07:37:27,633 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1560 - Command completed on 2024-08-07 07:37:27.633261
2024-08-07 07:37:27,633 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : subsystem add
2024-08-07 07:37:27,634 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.2.17-8  --server-address 10.0.65.219 --server-port 5500 subsystem add  --subsystem nqn.2016-06.io.spdk:cnode1 --max-namespaces 32 on 10.0.65.219 timeout 600
2024-08-07 07:37:27,634 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : subsystem add
2024-08-07 07:37:27,635 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.2.17-8  --server-address 10.0.65.219 --server-port 5500 subsystem add  --subsystem nqn.2016-06.io.spdk:cnode2 --max-namespaces 32 on 10.0.65.219 timeout 600
2024-08-07 07:37:28,865 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:28,865 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Adding subsystem nqn.2016-06.io.spdk:cnode2: Successful\n')
2024-08-07 07:37:28,865 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : gw info
2024-08-07 07:37:28,866 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.2.17-8  --format json --server-address 10.0.65.219 --server-port 5500 gw info  on 10.0.65.219 timeout 600
2024-08-07 07:37:29,043 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:29,043 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Adding subsystem nqn.2016-06.io.spdk:cnode1: Successful\n')
2024-08-07 07:37:29,044 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : gw info
2024-08-07 07:37:29,044 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.2.17-8  --format json --server-address 10.0.65.219 --server-port 5500 gw info  on 10.0.65.219 timeout 600
2024-08-07 07:37:29,918 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:29,918 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', '{\n    "cli_version": "1.2.17",\n    "version": "1.2.17",\n    "name": "client.nvmeof.rbd.ceph-nvme-1-wzk17m-node6.gllzxo",\n    "addr": "10.0.65.219",\n    "port": "5500",\n    "bool_status": true,\n    "error_message": "Success",\n    "spdk_version": "24.01.1",\n    "load_balancing_group": 1,\n    "hostname": "ceph-nvme-1-wzk17m-node6",\n    "group": "",\n    "status": 0\n}\n')
2024-08-07 07:37:29,918 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : listener add
2024-08-07 07:37:29,919 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.2.17-8  --server-address 10.0.65.219 --server-port 5500 listener add  --host-name ceph-nvme-1-wzk17m-node6 --traddr 10.0.65.219 --trsvcid 5002 --subsystem nqn.2016-06.io.spdk:cnode2 on 10.0.65.219 timeout 600
2024-08-07 07:37:30,093 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:30,093 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', '{\n    "cli_version": "1.2.17",\n    "version": "1.2.17",\n    "name": "client.nvmeof.rbd.ceph-nvme-1-wzk17m-node6.gllzxo",\n    "addr": "10.0.65.219",\n    "port": "5500",\n    "bool_status": true,\n    "error_message": "Success",\n    "spdk_version": "24.01.1",\n    "load_balancing_group": 1,\n    "hostname": "ceph-nvme-1-wzk17m-node6",\n    "group": "",\n    "status": 0\n}\n')
2024-08-07 07:37:30,093 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : listener add
2024-08-07 07:37:30,093 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.2.17-8  --server-address 10.0.65.219 --server-port 5500 listener add  --host-name ceph-nvme-1-wzk17m-node6 --traddr 10.0.65.219 --trsvcid 5001 --subsystem nqn.2016-06.io.spdk:cnode1 on 10.0.65.219 timeout 600
2024-08-07 07:37:30,935 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:30,935 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Adding nqn.2016-06.io.spdk:cnode2 listener at 10.0.65.219:5002: Successful\n')
2024-08-07 07:37:30,936 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command yum install -y nvme-cli fio on 10.0.64.123 timeout 600
2024-08-07 07:37:31,171 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:31,171 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Adding nqn.2016-06.io.spdk:cnode1 listener at 10.0.65.219:5001: Successful\n')
2024-08-07 07:37:31,171 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : host add
2024-08-07 07:37:31,171 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.2.17-8  --server-address 10.0.65.219 --server-port 5500 host add  --subsystem nqn.2016-06.io.spdk:cnode1 --host '*' on 10.0.65.219 timeout 600
2024-08-07 07:37:32,134 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:32,135 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Allowing open host access to nqn.2016-06.io.spdk:cnode1: Successful\n')
2024-08-07 07:37:32,135 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command rbd create rbd/9LJ2-image0 --size 10G on 10.0.64.123 timeout 600
2024-08-07 07:37:32,278 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:32,278 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.rbd.rbd_utils.py:107 - Command execution complete
2024-08-07 07:37:32,278 (cephci.test_ceph_nvmeof_gateway) [DEBUG] - cephci.parallel.py:89 - result is 0
2024-08-07 07:37:32,279 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : namespace add
2024-08-07 07:37:32,279 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.2.17-8  --server-address 10.0.65.219 --server-port 5500 namespace add  --subsystem nqn.2016-06.io.spdk:cnode1 --rbd-pool rbd --rbd-image 9LJ2-image0 on 10.0.65.219 timeout 600
2024-08-07 07:37:33,076 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command modprobe nvme-fabrics on 10.0.64.123 timeout 600
2024-08-07 07:37:33,096 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command nvme show-hostnqn on 10.0.64.123 timeout 600
2024-08-07 07:37:33,115 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:33,116 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : host add
2024-08-07 07:37:33,116 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.2.17-8  --server-address 10.0.65.219 --server-port 5500 host add  --subsystem nqn.2016-06.io.spdk:cnode2 --host nqn.2014-08.org.nvmexpress:uuid:a92ca03f-7c47-4940-adc2-483468d0326c on 10.0.65.219 timeout 600
2024-08-07 07:37:33,288 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:33,288 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Adding namespace 1 to nqn.2016-06.io.spdk:cnode1: Successful\n')
2024-08-07 07:37:33,289 (cephci.test_ceph_nvmeof_gateway) [DEBUG] - cephci.parallel.py:89 - result is ('', 'Adding namespace 1 to nqn.2016-06.io.spdk:cnode1: Successful\n')
2024-08-07 07:37:34,051 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:34,052 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Adding host nqn.2014-08.org.nvmexpress:uuid:a92ca03f-7c47-4940-adc2-483468d0326c to nqn.2016-06.io.spdk:cnode2: Successful\n')
2024-08-07 07:37:34,052 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command rbd create rbd/T09U-image0 --size 10G on 10.0.64.123 timeout 600
2024-08-07 07:37:34,165 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:34,166 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.rbd.rbd_utils.py:107 - Command execution complete
2024-08-07 07:37:34,166 (cephci.test_ceph_nvmeof_gateway) [DEBUG] - cephci.parallel.py:89 - result is 0
2024-08-07 07:37:34,166 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:16 - NVMe CLI command : namespace add
2024-08-07 07:37:34,166 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1596 - Running command podman run --quiet --rm cp.stg.icr.io/cp/ibm-ceph/nvmeof-cli-rhel9:1.2.17-8  --server-address 10.0.65.219 --server-port 5500 namespace add  --subsystem nqn.2016-06.io.spdk:cnode2 --rbd-pool rbd --rbd-image T09U-image0 on 10.0.65.219 timeout 600
2024-08-07 07:37:35,158 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.ceph.py:1630 - Command completed successfully
2024-08-07 07:37:35,158 (cephci.test_ceph_nvmeof_gateway) [INFO] - cephci.nvmegw_cli.execute.py:36 - ('', 'Adding namespace 1 to nqn.2016-06.io.spdk:cnode2: Successful\n')
2024-08-07 07:37:35,158 (cephci.test_ceph_nvmeof_gateway) [DEBUG] - cephci.parallel.py:89 - result is ('', 'Adding namespace 1 to nqn.2016-06.io.spdk:cnode2: Successful\n')
```

**Pull request closed without merging**